shared/data/src/main/java/com/ivy/data/repository/ExchangeRatesRepository.kt

```diff
@@ -0,0 +1,29 @@
+package com.ivy.data.repository
+
+import com.ivy.data.db.entity.ExchangeRateEntity
+import com.ivy.data.model.ExchangeRate
+import com.ivy.data.remote.responses.ExchangeRatesResponse
+import kotlinx.coroutines.flow.Flow
+
+interface ExchangeRatesRepository {
+    val urls: List<String>
+
+    suspend fun fetchExchangeRates(): ExchangeRatesResponse?
```

Comment:
Do not expose this response. The repository should work only with domain models. Create a domain model using our exact types (e.g. `AssetCode`, `PositiveDouble`) and map dtos -> domaij models

Reply:
The `ExchangeRatesResponse` should not be exposed but we can fix that in a follow-up PR. This response is Dto and the repo should return only domain model